src/Microsoft.ML.Recommender/RecommendUtils.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Utilities;
+
+namespace Microsoft.ML.Runtime.Recommender
+{
+   internal static class RecommendUtils
```

Comment:
(nit) the whitespace is off here. Missing a space.

Reply:
Fixed. Good catch.